src/bindings/python/src/compatibility/openvino/__init__.py

```diff
@@ -47,6 +47,9 @@
     from openvino.runtime import save_model
     from openvino.runtime import layout_helpers
 
+    # Import backend for torch compile
+    from openvino.frontend.pytorch.torchdynamo import backend
```

Comment:
the same question - will it fail if torch is not installed?

Reply:
Hey Ilya, if torch is not installed and we import openvino, it doesn't throw an error. But if someone wants to use torch.compile they have to install torch anyway so it will work. 

This is to give the user a better experience, instead of importing the full openvino.frontend.pytorch.torchdynamo.backend

Reply:
Looks like `openvino.backend` is not self descriptive

Reply:
I don't see the difference with another PR where we had the discussion. My concern is that it will `import torch` even if user doesn't want to import torch. Importing torch is not free, this is an unnecessary import that we start- to make each time when using `import openvino` in case `torch` is installed. This is not an error, but it is not needed additional dependency in runtime. How can we avoid it?

I tend to think that we cannot make torch auto-registration via `import openvino` because of those technical difficulties (explained here and in the PR where it was discussed for the first time) and we should have an extra import for torch backend with a nice path like `import openvino.torch` but it shouldn't be imported here automatically.

Reply:
@suryasidd, is there other products that tries to implement the same registration schema, or using a callable as a backed argument in `torch.compile` is intended solution in this case when we cannot make registration as a PR in torch?

Reply:
Hey Sergey, when you say a nice path like openvino.torch, do you mean adding a folder under openvino called torch and having an init file there ?

Reply:
I haven't see any other prodcuts using callable yet. Will see if there are other ways they are registering the backedn

Reply:
Hey Sergey, added the changes as requested. Now we can import openvino.torch and it will import the backend.
It fails if torch is not installed which is the expected behaviour. I added a requirements.txt and in the documentation we can add instructions to do a pip install on this requirements.txt file